sklearn/tree/_criterion.pyx

```diff
@@ -356,23 +349,20 @@ cdef class ClassificationCriterion(Criterion):
         self.weighted_n_left = 0.0
         self.weighted_n_right = self.weighted_n_node_samples
 
-        cdef SIZE_t n_outputs = self.n_outputs
-        cdef SIZE_t* n_classes = self.n_classes
-        cdef SIZE_t label_count_stride = self.label_count_stride
-        cdef double* label_count_total = self.label_count_total
-        cdef double* label_count_left = self.label_count_left
-        cdef double* label_count_right = self.label_count_right
+        cdef double* node_sum_total = self.node_sum_total
+        cdef double* node_sum_left = self.node_sum_left
+        cdef double* node_sum_right = self.node_sum_right
 
-        cdef SIZE_t k = 0
+        cdef SIZE_t k
 
-        for k in range(n_outputs):
-            memset(label_count_left, 0, n_classes[k] * sizeof(double))
-            memcpy(label_count_right, label_count_total,
-                   n_classes[k] * sizeof(double))
+        for k in range(self.n_outputs):
+            memset(node_sum_left, 0, self.n_classes[k] * sizeof(double))
+            memcpy(node_sum_right, node_sum_total,
+                   self.n_classes[k] * sizeof(double))
 
-            label_count_total += label_count_stride
-            label_count_left += label_count_stride
-            label_count_right += label_count_stride
+            node_sum_total += self.label_count_stride
+            node_sum_left += self.label_count_stride
+            node_sum_right += self.label_count_stride
```

Comment:
Shouldnt you also rename `label_count_stride` for consistency with your new names?